views/components/alert.html

```diff
@@ -1,3 +1,5 @@
-{% macro alert(for, type, value) %}
-<p class="alert--{{type | upper}}" role="alert" for="{{for}}">{{value}}</p>
+{% macro alert(data) %}
+{% for item in data %}
+<p class="form--alert" role="alert" for="{{item.for}}"><strong class="form--alert-type">{{item.type}} </strong>{{item.value}}</p>
```

Comment:
Currently, all alerts look the same and have the same level of visual importance. I was attempting to add some distinction between different alerts by adding the `type`. I will remove for now and create an issue to explore this idea in the future.

<img width="522" alt="screen shot 2016-12-08 at 3 52 53 pm" src="https://cloud.githubusercontent.com/assets/3981381/21029265/7da2ca3e-bd5e-11e6-95c2-4a7958918c1f.png">